api/serializers.py

```diff
@@ -62,6 +62,20 @@ def create(self, validated_data):
         return user
 
 
+class UserSerializerWithAssets(UserSerializer):
```

Comment:
@collinmutembei , Including the field inside the `UserSerializer` was the initial approach but it resulted into some errors (Circular dependency) IMHO since `get_allocated_assets` method has an attribute `serialized_assets = AssetSerializer(assets, many=True)`  and the `AssetSerializer` inturn has this field `assigned_to = UserSerializer(read_only=True)`

Reply:
🤔 I see; in that case...let's work with it as it is.